pkg/kubectl/cmd/cmd.go

```diff
@@ -183,11 +183,12 @@ __custom_func() {
 
     * all
     * certificatesigningrequests (aka 'csr')
-    * clusters (valid only for federation apiservers)
     * clusterrolebindings
     * clusterroles
+    * clusters (valid only for federation apiservers)
     * componentstatuses (aka 'cs')
     * configmaps (aka 'cm')
+    * cronjobs
```

Comment:
I'd like we also put a short name for this, such as `cjobs` etc. But we can fix this in a follow up patch.